src/main/java/service/ProductService.java

```diff
@@ -0,0 +1,9 @@
+package service;
+
+import com.es.phoneshop.exceptions.ProductNotFoundException;
+import com.es.phoneshop.model.product.ProductDao;
+
+public interface ProductService {
+    void init() throws ProductNotFoundException;
+    ProductDao getProductDao();
```

Comment:
no getter for dao is needed for access from servlet
but instead of getter methods delete(), save(), findProducts(), getProduct() are needed